intro-HPC/ch_multi_job_submission.tex

```diff
@@ -130,7 +130,7 @@ \section{The worker Framework: Parameter Sweeps}
 The job can now be submitted as follows:
 
 \begin{prompt}
-%\shellcmd{module load worker}%
+%\shellcmd{module load worker/1.6.8-intel-2018a}%
```

Comment:
@redfast00 Is it mentioned somewhere how to check which `worker` modules are available (since what you mention here is site-specific)?